Fabric.Authorization.API/scripts/Install-Authorization-Utilities.psm1

```diff
@@ -270,6 +270,43 @@ function Add-RoleToGroup
     Invoke-Sql $connString $query @{groupId = $groupId; roleId = $roleId; ; clientId = $clientId} | Out-Null
 }
 
+function Add-UserOrGroupToEdwAdmin($userOrGroup, $connString) {
+	$query   = "DECLARE @userId INTEGER;
+                DECLARE @roleId INTEGER;
+                
+                SET @userId = -1
+                SELECT @userId = COALESCE([IdentityID], -1) FROM [EDWAdmin].[CatalystAdmin].[IdentityBASE] WHERE [IdentityNM] = @identityName
+                BEGIN TRY
+                BEGIN TRANSACTION
+                    IF (@userId < 0)
+                    BEGIN
+                        INSERT INTO [EDWAdmin].[CatalystAdmin].[IdentityBASE] ([IdentityNM])
+                        VALUES (@identityName);
+                        SELECT @userId = SCOPE_IDENTITY();
+                    END
+                
+                    SELECT @roleId = [RoleID]
+                    FROM [EDWAdmin].[CatalystAdmin].[RoleBASE]
+                    WHERE [RoleNM] = @roleName;
```

Comment:
global?

Reply:
This is a sql parameter name

Reply:
Yah I figure it out by looking at the invoke where the -parameters are declared/defined.